### PR TITLE
feat: integrated localization

### PR DESCRIPTION
**Pull request by rumoursbehindme**

### CODE REVIEW

File: src/components/sidebar/index.tsx [hunk 1]
```diff
@@ -4,14 +4,16 @@ import SidebarButton from './sidebar-button';
 import { MdSpaceDashboard, MdFavorite } from "react-icons/md";
 import { FaGripfire, FaPlay, FaSignOutAlt } from 'react-icons/fa';
 import { IoLibrary } from "react-icons/io5";
-
+import { getTranslater } from '../../common/helper/localization-helper';
```
Comment (rumoursbehindme) [inline src/components/sidebar/index.tsx:7]:
```suggestion
import { getTranslater } from '../../common';
```

File: src/components/sidebar/index.tsx [hunk 2]
```diff
@@ -4,14 +4,16 @@ import SidebarButton from './sidebar-button';
...
 export default function SideBar() {
+
+    const t = getTranslater('sidebar')
     return (
         <div className='sidebar-container'>
             <img src='logo192.png' className='profile-img' alt='profile' />
             <div className='navigation-container'>
-                <SidebarButton title='Feed' redirectUrl='/feed' icon={<MdSpaceDashboard />} />
+                <SidebarButton title={t('feed')} redirectUrl='/feed' icon={<MdSpaceDashboard />} />
                 <SidebarButton title='Trending' redirectUrl='/trending' icon={<FaGripfire />} />
```
Comment (rumoursbehindme) [inline src/components/sidebar/index.tsx:17]: Add translation for each

**Comment by rumoursbehindme** [comment]
@HegdeParikshith  review madi bro

File: src/components/sidebar/index.tsx (hunk 2 above)
Comment (rumoursbehindme) [reply src/components/sidebar/index.tsx]: Fixed

**Review by Yashwanth-G22 — APPROVED** [review]

**Pull request merged**